SOLUTIONS-InspectingACandidateList.ipynb

```diff
@@ -0,0 +1,606 @@
+   "metadata": {},
+   "source": [
+    "## 5. Search the NAVO Registry for image resources that will allow you to search for AllWISE images.\n",
+    "\n",
+    "There are hundreds of image resources...too many to quickly read through. Try adding the 'keyword' search parameter to your registry search, and find the image resource you would need to search the AllWISE images."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "allwise_image_services = vo.regsearch(servicetype='image', keywords=['allwise'])#, 'NASA/IPAC Infrared Science Archive']) #, waveband='uv')\n",
+    "\n",
+    "\n",
+    "print(f'{len(allwise_image_services)} result(s) found.')\n",
+    "\n",
+    "allwise_image_services.to_table()['ivoid', 'short_name', 'res_title']\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6. Choose the AllWISE image service that you are interested in."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "allwise_image_service = allwise_image_services[0]\n",
+    "allwise_image_service.service"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 7. Choose one of the galaxies in the NED list.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.coordinates import SkyCoord\n",
+    "ra = galaxies['RA'][0]\n",
+    "dec = galaxies['DEC'][0]\n",
+    "pos = SkyCoord(ra, dec, unit = 'deg')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 8. Search for a list of AllWISE images that cover this galaxy.\n",
+    "\n",
+    "How many images are returned? Which are you most interested in?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "allwise_image_table = allwise_image_service.search(pos=pos, size=0)\n",
+    "allwise_image_table"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 9. How many images are returned for this one position? Which are you most interested in?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "allwise_images = allwise_image_table.to_table()\n",
+    "allwise_images"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 10. Filter the list of returned AllWISE images so that you are left with only the images taken in band W1 (3.6 micron)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for allwise_image_record in allwise_image_table:\n",
+    "    if 'W1' in allwise_image_record.bandpass_id:\n",
+    "        break"
```

Comment:
Add some sort of output? Print allwise_image_record?

Reply:
Excellent suggestion. I'll print `.title` and `.bandpass_id` for the selected record, here and for GALEX and Sloan.